nemo_skills/evaluation/evaluator/dsbench.py

```
@@ -0,0 +1,82 @@
+    """
+    Relaxed equality check with:
+    1. Case-insensitive MCQ matching
+    2. Dict/list comparison using math_equal recursively
+    """
+    if predicted_answer is None:
+        return gt_answer is None
+
+    try:
+        predicted_answer = json.loads(predicted_answer)
+        gt_answer = json.loads(gt_answer)
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)